etna/packages/etna-py/src/mountetna/metis.py

```diff
@@ -503,8 +504,9 @@ def upload_parts(
                 )
 
                 unsent_zero_byte_file = False
-            except HTTPError as e:
+            except RequestException as e:
                 if remaining_attempts > 1:
+                    sleep(30)
```

Comment:
checkout `gater.py`.  I believe I already have exponential backoff implemented there?  In any case, I think we want exponential back off starting with a lower threshold than 30 seconds.  But this is totally fine as a start.